### ISSUE TITLE
Please release code PU-VoxelNet

### ISSUE THREAD

**Issue by doldolOuO** [hikvision-research/3DVision#3]
What a nice work! Please open the source code of PU-VoxelNet(AAAI-2024). Thanks

**Comment by Alias-wzp** [comment]
agree.Im looking forward to get the source code of PU-VoxelNet(AAAI-2024).

**Comment by Alias-wzp** [comment]
Agree! Im looking forward to the open of the source code of PU-VoxelNet(AAAI-2024).
